app/register.js

```diff
@@ -33,20 +33,20 @@ export default function Login() {
 		<>
 			<StyledSafeArea
 				className="bg-offblack"
-				style={{ flex: 1 /*  backgroundColor: "#5946B2"  */ }}
+				style={{ flex: 1, backgroundColor: "#5946B2" }}
 			>
 				<KeyboardAwareScrollView>
 					<TouchableWithoutFeedback onPress={Keyboard.dismiss}>
 						<>
-							<StyledView className="flex flex-col pt-10 pb-5 px-[15px] items-center justify-start w-screen ">
-								<StyledView className="w-full flex flex-col items-center justify-center mb-2">
-									<StyledText className="text-offwhite text-center text-5xl font-bold">
-										Welcome to Prayer Circle!!
-									</StyledText>
-									<StyledView className="w-[40%] aspect-square my-2">
+							<StyledView className="flex flex-col pb-5 px-[15px] w-screen">
+								<StyledView className="w-full flex flex-col items-center mb-2">
+									{/* <StyledText className="text-offwhite text-center text-5xl font-bold">
+										Welcome to Prayer Circle!
+									</StyledText> */}
+									<StyledView className="w-[89%] aspect-square mt-[15%] mb-[10%]">
```

Comment:
not sure percentages are the best for margin. if we go with this, we need to test on different screen sizes to make sure everything looks good. I would suggest using static values, even if we use our own values(not the prebuilt ones): `m-[10px]`